File: College/C/1ª Lista de Exercícios (1S)/Exercício03.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int metros, milimetros;

    printf("Digite o valor em metros: ");
    scanf("%d", &metros);

    milimetros = metros * 100;

    printf("%d metro(s) é igual a %d milimetros.\n", metros, milimetros);

}

File: Java/1ª Lista de Exercícios (1S)/Exercício03/Exercício03.java
import java.util.Scanner;

public class Exercício03{
	public static void main(String args[]){
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite um valor em metros: ");
		int metros = scan.nextInt();

		int milimetros = metros * 100;

		System.out.println(metros + " metro em milimetros é igual a " + milimetros);

		scan.close();
	}
}
